VARIABLE.md

```diff
@@ -0,0 +1,128 @@
+# VARIABLES
+
+Variable is used to label and store data in computer memory.
+Variable can update information stored in variable,
+reference or get information stored in a variable.
+
+Its important to distinguish that variable are not value, they contain value and represent them.
+
+`Let` and `const` are new keyword to declare a variable, introduced in `ES6`. Prior to that programmers can declare variable with only the `var` keyword. example:
+
+```js
+var food = 'pizza';
+console.log(food); // output: pizza
+```
+
+`let` keyword signal that values can be reassigned a different value e.g
+
+```js
+let myName = 'Azeez';
+console.log(myName); //output: Azeez
+myName = 'Rotimi';
+console.log(myName); //output: Rotimi
+
+```
+
+If the `let` variable is declared without a value, the variable will be automatically initialized with a value of `undefined` example:
+
+```js
+let myAge;
+console.log(myAge); // output: undefined.
+```
+
+
+The `let` keyword is similar to using the `var` keyword because, anything that can be done with `var` can be done with `let`.
+
+The `const` keyword however cannot because its constant. If you try reassigning a `const` variable, a `TypeError`, will be thrown. example:
+
+```js
+const myCity = "Lagos";
+console.log(myCity); // output: Lagos
+myCity = "Ibadan";
+console.log(myCity); // output: TypeError
+```
+
+If a `const` variable is not assigned any value, a `SyntaxError`would be thrown. example:
+
+```js
+const myAge;
+console.log(myAge); // output: SyntaxError
```

Comment:
I think the error would have taken place in line 48 before line 49, See example on `node`
```js
const bola;
Thrown:
const bola;
      ^^^^

SyntaxError: Missing initializer in const declaration

```

Reply:
Fixed